scripts/unused_images.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
```suggestion

set -e -u -o pipefail

```

Would you consider adding these to ensure that errors are raised immediately and that unset variables aren't silently treated as empty strings?